code/modules/mob/living/carbon/human/species.dm

```diff
@@ -574,6 +578,7 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 
 	if(!hair_hidden || dynamic_hair_suffix)
 		var/mutable_appearance/hair_overlay = mutable_appearance(layer = -HAIR_LAYER)
+		var/mutable_appearance/gradient_overlay = mutable_appearance(layer = -HAIR_LAYER)
```

Comment:
Rather than a separate gradient overlay like this you could just be adding an overlay to the hair overlay

Reply:
Discussed in coderbus and some oddities with MA makes this fine.